packages/blade/src/components/ActionList/ActionList.tsx

```diff
@@ -127,9 +111,7 @@ const _ActionList = ({ children, surfaceLevel = 2, testID }: ActionListProps): J
           ref={actionListItemRef as any}
         />
       ) : (
-        <StyledActionList
-          isInBottomSheet={isInBottomSheet}
-          surfaceLevel={surfaceLevel}
```

Comment:
Why did we removed this?

Reply:
the styles of overlay are not on ActionList anymore. They are on `DropdownOverlay`. Changed this StyledActionList into just `BaseBox`